chrome-extension/content.js

```diff
@@ -162,26 +181,31 @@ async function getCategoryDropdown(userId) {
       categoryDropdown.options.add(newOption);
     });
   } catch (error) {
-    // TODO: Handle this error?
+    // TODO: Handle error
   }
   return categoryDropdown;
 }
 
 /**
  * Add new item to databse
```

Comment:
nit: "database" and update jsdoc to reflect the UI changes happening

Reply:
okay!